app/js/controllers/themes.js

```diff
@@ -0,0 +1,29 @@
+/* global React, ReactDOM */
+
+import { Controller } from 'components/fxos-mvc/dist/mvc';
+
+import Themes from 'js/views/themes';
+import ThemesNew from 'js/views/themes-new';
+
+export default class ThemesController extends Controller {
+  main(action = '') {
```

Comment:
__question:__ mm can't we just remove this default value and use ```default``` switch branch instead of weird ```case ''```?